### DISCUSSION TITLE
Uninstall InstallShield Before Running Wix Install

### DISCUSSION THREAD

**Discussion by KMarois** [wixtoolset/issues#7207] [Questions]
I am building a Wix project as a replacement for an Installshield project. Is it possible to have Wix uninstall the previous InstallShield install first?

Would setting the UpgradeCode to the GUID in InstallShield do the trick?

Thanks

**Comment by woscostametaxa** [comment]
If the UpgradeCode matches, and the new WiX version is higher - an update should uninstall the installshield msi. 

If you are talking about a WixProject just to create an Msi (or did you mean something like a Burn bootstrapping application?)
